### PR TITLE
WooCommerce Install: Update styles

### PR DESCRIPTION
**Pull request by obenland**
Brings styles closer to the figma file.

#### Changes proposed in this Pull Request

* Moves styles to a central file
* Establishes a mixing for two-column layout that can be used by onboarding steps.
* Updates form element styles to correspond with the figma design.
* Moves two elements into the woocommerce-install/index file since they're now used by all steps.

#### Testing instructions

- Go to http://calypso.localhost:3000/woocommerce-installation/
- Choose a test site site
- Click "Set up my store"
- Navigate through the onboarding steps and make sure the form elements look like in the figma
- Check that the transfer screen still looks the same.

#### After

Screen Shot 2022-01-18 at 3 57 28 PM

Screen Shot 2022-01-18 at 3 57 33 PM

### CODE REVIEW

**Comment by obenland** [comment]
Somehow I broke the transfer step styles with this. So I'll look at that next.

File: client/signup/steps/woocommerce-install/step-store-address/style.scss [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
 	.components-form-token-field__suggestions-list {
+		margin: 4px -16px -12px;
 		width: min-content;
 	}
 
 	.components-combobox-control__reset.components-button {
-		// Height of 22 ensures combobox input matches height of text controls
-		height: 22px;
+		// Height of 18 ensures combobox input matches height of text controls.
+		height: 18px;
```
Comment (lsl) [inline client/signup/steps/woocommerce-install/step-store-address/style.scss:23]: Probably still want to go ahead with this change here but FYI a fix might come from upstream at some point https://github.com/WordPress/gutenberg/pull/38020

**Comment by Rolilink** [comment]
Maybe this is me being picky but the label colors, I think don't match the ones in the design

**Comment by obenland** [comment]
> Maybe this is me being picky but the label colors, I think don't match the ones in the design

That should be a matter of which color scheme you use, I don't believe I'm currently using the default one.

**Review by DustyReagan — APPROVED** [review]
I think I see what @Rolilink is talking about regarding the font colors. The Figma looks slightly darker, but the hex values appear to be the same. I think it could be a minor difference in font or possibly the field borders making the font color appear lighter. But I think this PR is good to ship. 👍 

After pics:

**Pull request merged**